tests/integration/models/keys.spec.coffee

```diff
@@ -76,7 +76,9 @@ describe 'Key Model', ->
 
 			it 'should be rejected if the key id is invalid', ->
 				promise = resin.models.key.get(99999999999)
-				m.chai.expect(promise).to.be.rejectedWith('Request error: Internal Server Error')
+				m.chai.expect(promise).to.be.rejectedWith('Request error: value "99999999999" is out of range for type integer')
```

Comment:
Yeah, you're right, let's simplify it to just look for a request error generally.

I don't think we need the historical comments below either, since all this info is in the commit log. I'd just drop them.